src/modules/VirtConfig.rb

```diff
@@ -317,6 +318,70 @@ def LXCDialog
         )
       end
 
+      Builtins.y2milestone("VirtConfig::ConfigureDom0: Checking for Installed Patterns and Packages")
+      is_xen_server_installed = false
+      is_xen_tools_installed = false
+      is_kvm_server_installed = false
+      is_kvm_tools_installed = false
+      is_client_tools_installed = false
+      if isOpenSuse == true
```

Comment:
The `isOpenSuse` and other `is*` methods return boolean, you can write simple `if isOpenSuse` in this case, no need to compare with `true`.